apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -220,7 +220,10 @@ export class BulkScriptAction extends BaseScriptAction {
    */
   private async _runWatch(options: IExecuteInternalOptions): Promise<void> {
     const {
-      taskSelectorOptions: { selection: projectsToWatch },
+      taskSelectorOptions: {
+        buildCacheConfiguration: initialBuildCacheConfiguration,
+        selection: projectsToWatch
```

Comment:
I thought `selection` was going to be renamed in the `IExecuteInternalOptions` type?

Reply:
This is `ITaskSelectorConstructor`. The rename was in `IProjectWatcherOptions`. This value is the set of projects to invoke the command on for single execution.